sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -4,41 +4,73 @@
 
 ```ts
 
+import * as coreClient from '@azure/core-client';
+import * as coreHttps from '@azure/core-rest-pipeline';
```

Comment:
did we intend to alias this as a group? I notice we directly import `PipelineOptions` below

Reply:
I believe these two are from the generated code and probably pulled in by api-extractor when we export generated types.

Reply:
Somehow it's gone in the latest version, after I removed export of a generated type.